openlibrary/templates/type/delete/view.html

```diff
@@ -12,7 +12,7 @@ <h1>$_("This page has been deleted")</h1>
     <p><strong>$_("What would you like to do?")</strong></p>
 
     <ul>
-      <li><a href="javascript:;" onclick="history.go(-1);">$_("Go back where I came from")</a></li>
+      <li><a href="javascript:;" id="go-back">$_("Go back where I came from")</a></li>
```

Comment:
```suggestion
      <li><a href="javascript:;" class="go-back-link">$_("Go back where I came from")</a></li>
```
There are a number of these links throughout the codebase.  I think that it's best to identify them with a common class, then add the click listener to all elements having this class.

Doing a quick search for `history.go` with my IDE yielded 14 matches.  I'd expect all of these to be updated to use your click listener in this branch.